samples/peripheral/radio_test/README.rst

```diff
@@ -0,0 +1,149 @@
+.. _radio_test:
+
+Radio Test
+##########
+
+The Radio Test sample demonstrates how to configure the radio as a constant RX or TX carrier, a modulated TX carrier, or a RX or TX sweep.
+
+Overview
+********
+
+The tests are controlled with Command Line Interface (CLI) through the serial port.
+At any time during the tests, you can set the radio parameters, such as output power, bit rate, and channel.
+You can also set the time on each channel in the sweep mode, in steps of 1 millisecond to 99 milliseconds, every 1 millisecond.
+The sample also allows you to send a data pattern to another board.
+
+The sample starts with enabling the high frequency crystal oscillator and configuring CLI.
+Running a console command causes the appropriate action.
```

Comment:
Can we be a bit more descriptive here?
Something like:
The sample then waits for user input and executes the entered commands (to configure the radio?).
See `User Interface`_ for a list of available commands.